### ISSUE TITLE
Project abandoned? / Offer to take it over

### ISSUE THREAD

**Issue by monarchjhaveri** [zippyui/react-datagrid#269]
Looking at the number of issues that are open on this project and on https://github.com/zippyui/react-date-picker, it looks like the author has abandoned these projects.

Dear author: Is this the case? If you are no longer interested in maintaining these projects, can you please contact me and perhaps I can take over maintaining them for you?

**Comment by radubrehar** [comment]
No, not abandoned! We're just preparing something much bigger! :)

**Comment by monarchjhaveri** [comment]
Great to hear! Looking forward to it. You might want to change the messaging on the Readme because a lot of people are getting the idea that this project is abandoned :-)

**Comment by nhpace** [comment]
Can you elaborate on the big changes? Some of us depend on this project in its current form. It would be nice to know: are you going to continue to enhance and support this project or are you moving onto something new?

**Comment by monarchjhaveri** [comment]
Reopened the issue since people seem to want more answers :)

i took a brief look at http://zippyui.com/, looks like some sort of components package or framework is the order of the day

this project needs to be supported regardless, abandoned projects are bad publicity :-)

**Comment by radubrehar** [comment]
We're going to continue to support those components in the new releases.

**Comment by nhpace** [comment]
I'm sorry but your answer is a little vague to me. Will you be continuing any activity on react-datagrid, specifically? If not, would you consider allowing someone else to maintain it?

**Comment by monarchjhaveri** [comment]
When will the new releases be coming out?
Are you able to prioritize and tackle some of the more critical issues here?

**Comment by nhpace** [comment]
Any updates? If not, no choice but to assume this project has been abandoned. If abandoned, it would be difficult to use any future projects by this contributor.
